plugins/woocommerce-beta-tester/api/live-branches/install.php

```diff
@@ -0,0 +1,95 @@
+<?php
+/**
+ * Register REST endpoint for installing live branches manifest.
+ *
+ * @package WC_Beta_Tester
+ */
+
+require_once __DIR__ . '/../../includes/class-wc-beta-tester-live-branches-installer.php';
+
+register_woocommerce_admin_test_helper_rest_route(
+	'/live-branches/install/v1',
+	'install_version',
+	array(
+		'methods' => 'POST',
+	)
+);
+
+register_woocommerce_admin_test_helper_rest_route(
+	'/live-branches/deactivate/v1',
+	'deactivate_woocommerce',
+	array(
+		'methods' => 'GET',
+	)
+);
+
+register_woocommerce_admin_test_helper_rest_route(
+	'/live-branches/activate/v1',
+	'activate_version',
+	array(
+		'methods'             => 'POST',
+		'permission_callback' => function( $request ) {
+			// Avoid using WC functions as core will be deactivated during this request.
```

Comment:
When we go to activate a plugin version, there may not be an active version of WooCommerce installed. By overriding the permission_callback here and using core WP permission checks we ensure that the site does not crash during activation.